librz/core/cmd/cmd_search_rop.c

```diff
@@ -10,11 +10,27 @@
 #include "rz_rop.h"
 
 static void skip_whitespace(const char *str, ut64 *idx) {
+	if (*idx >= strlen(str)) {
```

Comment:
is there any possibility that `str` is NULL? if is not possible, ignore this.

Reply:
This is always called from this [code](https://github.com/giridharprasath/rizin/blob/3aeef0a25afd7a4747c86d1f9d262722d2deb1d2/librz/core/cmd/cmd_search_rop.c#L565) path